src/main/webapp/WEB-INF/views/car/display.jsp

```diff
@@ -0,0 +1,29 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Display all cars</title>
+</head>
+<body>
+<h1>List of cars:</h1>
+<table>
+    <tr>
+        <td>Car id</td>
+        <td>Model</td>
+        <td>Manufacturer name</td>
+        <td>Manufacturer country</td>
+        <td>Driver list</td>
+    </tr>
+    <c:forEach items="${cars}" var="car">
+        <tr>
+            <td><c:out value="${car.id}" /> </td>
+            <td><c:out value="${car.model}" /> </td>
+            <td><c:out value="${car.manufacturer.name}" /> </td>
+            <td><c:out value="${car.manufacturer.country}" /> </td>
+            <td><c:out value="${car.drivers}" /> </td>
```

Comment:
use <c:forEach> to display drivers properly. Create a separate column for name and license

Reply:
Im really agree with it, but dont know how to make it propertly. because it doesnt look so good